parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -247,6 +247,9 @@ else
 fi
 
 if [[ "${MESSAGE_OF_THE_DAY}" != "" ]]; then
+    if isMarinerOrAzureLinux "$OS"; then
+      sed -i "s/emit_via = motd/emit_via = stdio/g" /etc/dnf/automatic.conf
```

Comment:
hmm not a big fan on in file updates but I guess its fine, I see that we already check the file status in e2e, so its ok

Reply:
Context: https://github.com/microsoft/azurelinux/commit/56ffe4180550364cce739a07645539ebbd774eb8#diff-b95e1980d5c078a15548240d96a5ba7204b49926ac9303b0f8ca2dfbd4221a3eR55